tests/e2e/taqueria-plugin-contract-types.spec.ts

```diff
@@ -1,333 +1,289 @@
 const exec = util.promisify(exec1);
 
-const taqueriaProjectPath = 'scrap/auto-test-contract-types-plugin';
+import { prepareEnvironment } from '@gmrchk/cli-testing-library';
 
-describe('E2E Testing for taqueria contract types plugin only', () => {
-	beforeAll(async () => {
-		await generateTestProject(taqueriaProjectPath, ['contract-types']);
-		// TODO: This can removed after this is resolved:
-		// https://github.com/ecadlabs/taqueria/issues/528
-		try {
-			await exec(`taq -p ${taqueriaProjectPath}`);
-		} catch (_) {}
-	});
+describe('Contract Types Plugin E2E Testing for Taqueria CLI', () => {
+	test('contract types plugin will offer help', async () => {
+		const { execute, spawn, cleanup } = await prepareEnvironment();
+		const { waitForText } = await spawn('taq', 'init test-project');
+		await waitForText("Project taq'ified!");
+		const { stdout } = await execute('taq', 'install @taqueria/plugin-contract-types', './test-project');
+		expect(stdout).toContain('Plugin installed successfully');
 
-	// Skipping as help output has changed
-	test.skip('Verify that the contract types plugin exposes the associated commands in the help menu', async () => {
-		try {
-			const generateTypesHelpContents = await exec(`taq --help --projectDir=${taqueriaProjectPath}`);
-			expect(generateTypesHelpContents.stdout).toBe(contents.helpContentsContractTypesPlugin);
-		} catch (error) {
-			throw new Error(`error: ${error}`);
-		}
-	});
+		const { stdout: stdout2 } = await execute('taq', '--help --projectDir=./test-project');
+		expect(stdout2).toEqual(expect.arrayContaining(['Commands:']));
 
-	// Skipping as help output has changed
-	test.skip('Verify that the contract types plugin exposes the associated options in the help menu', async () => {
-		try {
-			const generateTypesHelpContents = await exec(`taq generate types --help --projectDir=${taqueriaProjectPath}`);
-			expect(generateTypesHelpContents.stdout).toBe(contents.helpContentsContractTypesPluginSpecific);
-		} catch (error) {
-			throw new Error(`error: ${error}`);
-		}
+		await cleanup();
 	});
 
-	// Skipping as help output has changed
-	test.skip('Verify that the contract types plugin exposes the associated aliases in the help menu', async () => {
-		try {
-			const generateTypesHelpContentsGen = await exec(`taq gen --help --projectDir=${taqueriaProjectPath}`);
-			expect(generateTypesHelpContentsGen.stdout).toBe(contents.helpContentsContractTypesPluginSpecific);
-
-			const generateTypesHelpContentsGenTypes = await exec(`taq gentypes --help --projectDir=${taqueriaProjectPath}`);
-			expect(generateTypesHelpContentsGenTypes.stdout).toBe(contents.helpContentsContractTypesPluginSpecific);
-		} catch (error) {
-			throw new Error(`error: ${error}`);
-		}
-	});
+	test.skip('1635 - generate types offers contextual help', async () => {
+		const { execute, spawn, cleanup } = await prepareEnvironment();
+		const { waitForText } = await spawn('taq', 'init test-project');
```

Comment:
Btw do you mean to install the "core" plugin from NPM or local? If local, then provide `--debug` flag. If not this is fine.

This applies to all `taq init`s in e2e tests.

Reply:
Note: "core" plugin is installed automatically upon `taq init`. That's why I wanted to confirm.

Reply:
The plugin loading will now be local only by relative path traversal